### DISCUSSION TITLE
Welcome to AsusS530UN-hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by tunglamvghy** [tunglamvghy/AsusS530UN-hackintosh#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luoling76** [comment]
你好，我的笔记本是ASUSvivobook S14 4300u 
CPU：i78550U
显卡：620
用你的EFI，出现睡眠不了，请问怎么解决这个问题

**Reply by tunglamvghy** [reply]
Do you understand English?
Because your laptop has different CPU, you need to disable CPUFriend and CPUFriendDataProvider kext. Does your laptop have nvidia graphics card?

**Reply by luoling76** [reply]
If I disable the above two kext the computer will not turn on

**Reply by luoling76** [reply]
mean, with your EFI, all the problems you list are working, that is, sleep doesn't work.

**Reply by tunglamvghy** [reply]
you can use this (https://github.com/stevezhengshiqi/one-key-cpufriend) to generate new CPUFriend kext for your laptop. Then try to put your pc to sleep

**Reply by luoling76** [reply]
> Do you understand English?
> Because your laptop has different CPU, you need to disable CPUFriend and CPUFriendDataProvider kext. Does your laptop have nvidia graphics card?
If I disable the above two kext the computer will not turn on
with your EFI, all the problems you list are working, that is, sleep doesn't work.

**Reply by luoling76** [reply]
I did, sleep or fail

**Comment by luoling76** [comment]
你好！首先感谢你的efi。
你用耳机时候会出现只有背景音乐。这个问题怎么办？

**Reply by tunglamvghy** [reply]
Go to Sound setting > Ouput, then choose Headphone. Adjust the Balance to the right or to the left until you get the good sound

**Reply by luoling76** [reply]
你有微信吗，加微信聊

**Reply by tunglamvghy** [reply]
> 你有微信吗，加微信聊

Sorry for answering late, I have Wechat account but I rarely use it. You can contact me via Telegram.

**Comment by zeekands** [comment]
hi, just wanna say thank you. i have an Asus a407uf with an i7-8550u processor, I have frustrated bout how to make EFI works on my laptop. then I find your EFI cause I think that our laptop has a similar specification. then boom it works perfectly good.

**Reply by tunglamvghy** [reply]
Remember to remap your usb ports and CPUFriend. That will make your pc better.

**Reply by zeekands** [reply]
Could you please help me to make it, I have no idea how to remap and edit cpufriend. Or maybe you have some tutorial that could help me.

**Reply by zeekands** [reply]
I'm newbie to this hackintosh and frustrated just to install that and make it work.

**Reply by tunglamvghy** [reply]
> I'm newbie to this hackintosh and frustrated just to install that and make it work.

1. Use hackintool and choose USB tab. Then use your usb plug all ports (you need a pendrive 2.0, a pendrive 3.0 and a typeC pendrive). Then delete all grey ports. Click Export, then copy USBPorts.kext in Desktop to /OC/Kexts
2. Search Google for CPUFriendFriend. It's very easy. It will generate a file named SSDT-DATA.aml. Overwrite the file in /OC/ACPI

**Reply by zeekands** [reply]
thank you for the guide, now i can set up my Hackintosh properly.

**Comment by superzeldalink** [comment]
Are you planning to update to Monterey beta?

**Reply by zeekands** [reply]
yea, i hope that you could update this efi to make it work for monterey

**Reply by luoling76** [reply]
我不准备更新12了

**Reply by tunglamvghy** [reply]
Sorry for answering late. I'm very busy with my exam so I cant spend time on Hackintosh.
On newer version of macOS, there are many errors that need to be fixed before you want to upgrade. Now, there is a problem with CpuTscSync kext that caused kernel panic while updating to macos 12. So, wait until the devs fix this error.

**Reply by tunglamvghy** [reply]
> yea, i hope that you could update this efi to make it work for monterey

A new version of EFI has just been published to support macOS 12. You can download it and try in branch "monterey" in the repo.

**Comment by Frankensten** [comment]
Is the latest version stable on macOS 11.4? I want to have a try with the lossless audio in Apple Music🤪

**Reply by luoling76** [reply]
在 11.4相当稳定，目前不建议升级12.

**Reply by tunglamvghy** [reply]
> 在 11.4相当稳定，目前不建议升级12.

You can upgrade to macOS 11.4 but dont update to macOS 12 due to an error in CpuTscSync kext that is still buggy on macOS 12.
When newer version is available,  I'll update the repo for everyone to try macOS 12.

**Reply by Frankensten** [reply]
> You can upgrade to macOS 11.4 but dont update to macOS 12 due to an error in CpuTscSync kext that is still buggy on macOS 12.
> 
> When newer version is available,  I'll update the repo for everyone to try macOS 12.

Great! I'm waiting for your exciting work🥳

**Reply by tunglamvghy** [reply]
> 在 11.4相当稳定，目前不建议升级12.

A new version of EFI has just been published to support macOS 12. You can download it and try in branch "monterey" in the repo.

**Reply by superzeldalink** [reply]
> > 在 11.4相当稳定，目前不建议升级12.
> 
> A new version of EFI has just been published to support macOS 12. You can download it and try in branch "monterey" in the repo.

Can I update directly from Big Sur?

**Reply by tunglamvghy** [reply]
> > > 在 11.4相当稳定，目前不建议升级12.
> > 
> > 
> > A new version of EFI has just been published to support macOS 12. You can download it and try in branch "monterey" in the repo.
> 
> Can I update directly from Big Sur?

Yes. I've updated diectly from BigSur 11.4

**Reply by superzeldalink** [reply]
> > > > 在 11.4相当稳定，目前不建议升级12.
> > > 
> > > 
> > > A new version of EFI has just been published to support macOS 12. You can download it and try in branch "monterey" in the repo.
> > 
> > 
> > Can I update directly from Big Sur?
> 
> Yes. I've updated diectly from BigSur 11.4

Perfecttt
I’ll try it next week. This week I need to use my laptop a quite a lot

**Comment by Frankensten** [comment]
airportitlwm has updated to 2.0alpha, added support for monterey 5 days ago~

**Comment by zeekands** [comment]
is the efi avaiable for monterey relase version now?

**Reply by tunglamvghy** [reply]
Yes. You can update to monterey release version

**Comment by luoling76** [comment]
谢谢了，我买了M1了



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Lâm ***@***.***&gt;; 
发送时间: 2021年11月10日(星期三) 下午4:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [tunglamvghy/AsusS530UN-hackintosh] Welcome to AsusS530UN-hackintosh Discussions! (Discussion #25)





 
Yes. You can update to monterey release version
 
&mdash;
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Chetan8624** [comment]
Hello,

I have Asus TUF FX505GD, I have installed macOS Monterey. Everything is working except that I want to configure keyboard light increase/decrease button and also some function keys. Additionally, I want to setup fan control as well.

Could you please help me with configuring function keys for keyboard light and etc.

**Reply by luoling76** [reply]
不好意思，我买了m1。

**Comment by stardustdev** [comment]
@tunglamvghy

My spec is Asus Vivobook s15 i7 11th.
https://www.dns-shop.ru/product/05068b0566c0ed20/156-noutbuk-asus-vivobook-s15-s533ea-bn313-belyj/characteristics/

Can I use your EFI for my laptop?

**Reply by tunglamvghy** [reply]
No. Laptop with 11th processor without supported external graphic card cannot install Hackintosh

**Comment by Exoalam** [comment]
how to solve usbtoolbox waitformatchingservices

**Reply by tunglamvghy** [reply]
> how to solve usbtoolbox waitformatchingservices

Follow this guide to create your own kext. 
[https://github.com/USBToolBox/tool](url)

**Reply by Exoalam** [reply]
My laptop is Asus S530F. Do I still need build kext? I am beginner in hackintosh.

**Reply by Exoalam** [reply]
usbtoolbox XHC: waitformatchingservice failed or timeout

**Reply by Exoalam** [reply]
I am getting this error at the installation of ventura. Do I need to change any bios settings? I followed you installation guide
